README.md

```diff
@@ -224,7 +224,49 @@ For more information: https://pmd.github.io/pmd-5.4.1/customizing/rule-guideline
 
 - Flooded console output when running Checkstyle in debug mode in Maven  (- X ) - https://github.com/checkstyle/checkstyle/issues/3184;
 
+# ESH Guidelines Covered
+
+## A. Code Style
+1. The Java naming conventions should be used. - https://github.com/openhab/static-code-analysis/blob/master/src/main/resources/rulesets/checkstyle/rules.xml#L80
+2. Every Java file must have a license header. You can run mvn license:format on the root of the repo to automatically add missing headers. - https://github.com/openhab/static-code-analysis/blob/master/src/main/resources/rulesets/checkstyle/rules.xml#L34
+3. Every class, interface and enumeration should have JavaDoc describing its purpose and usage. - https://github.com/openhab/static-code-analysis/blob/master/src/main/resources/rulesets/checkstyle/rules.xml#L66
+4. Every class, interface and enumeration must have an @author tag in its JavaDoc for every author that wrote a substantial part of the file. - `Work in Progress`
+5. Every constant, field and method with default, protected or public visibility should have JavaDoc (optional, but encouraged for private visibility as well). - `Not covered yet`
+6. Code must be formatted using the provided code formatter and clean up settings. They are set up automatically by the official IDE setup. - `Not covered yet`
```

Comment:
This should probably be split up - we have dozens of code formatter settings, and we won't (and don't want) to maintain them separately here as well in all details.

With #12, we will have the most common issue covered already - we should add a similar check for xml and json files (where we actually expect tabs, no spaces). Having these checks in place, I would consider A.6 to be checked well enough.